app/code/Magento/Store/Model/Service/StoreConfigManager.php

```diff
@@ -71,12 +82,14 @@ public function getStoreConfigs(array $storeCodes = null)
     }
 
     /**
-     * @param \Magento\Store\Model\Store $store
-     * @return \Magento\Store\Api\Data\StoreConfigInterface
+     * Get store specific configs
+     *
+     * @param Store|StoreInterface $store
+     * @return StoreConfigInterface
      */
-    protected function getStoreConfig($store)
+    public function getStoreConfig($store)
```

Comment:
I don't think we need this change, and I would prefer to avoid it

Reply:
I know we discussed this offline and thought it might not be a big deal.
However, now the classes api differs from it's interface, and updating the interface would be backwards incompatible.

Reply:
you can cache values in getStoreConfigs, so it fetches data just once
then you can call `getStoreConfigs([$store->getCode()])` instead of calling getStoreConfig($store)
It's the same thing